pkgs/development/ocaml-modules/ocamlfuse/default.nix

```diff
@@ -1,22 +1,17 @@
-{ stdenv, fetchFromGitHub, ocaml, camlidl, fuse, findlib }:
+{ stdenv, buildDunePackage, fetchFromGitHub, camlidl, fuse }:
 
-stdenv.mkDerivation rec {
+buildDunePackage {
   pname = "ocamlfuse";
-  version = "2.7.1_cvs5";
+  version = "e35e76b";
```

Comment:
Please find a better version number, possibly larger than `2.7.1_cvs6` and smaller than `2.7.2`

Reply:
The problem with making up our own is that it may be used by upstream in the future and refer to something else, especially `2.7.1_cvs7` or similar, but maybe `2.7.1_cvs6_e35e76b` could work?

Reply:
I changed it to `2.7.1_cvs6_e35e76b`.

Reply:
This seems a good choice. Thanks.